### PR TITLE
{bio}[GCCcore/8.2.0] FastTree v2.1.11

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Darkless012** [comment]
The code-style-error is about not having HTTPS source_urls, but the site doesn't have valid certificate for this site, can the check be disabled somehow?

File: easybuild/easyconfigs/f/FastTree/FastTree-2.1.11-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'CmdCp'
+
+name = 'FastTree'
+version = '2.1.11'
+
+homepage = 'https://www.microbesonline.org/fasttree/'
```
Comment (Micket) [inline easybuild/easyconfigs/f/FastTree/FastTree-2.1.11-GCCcore-8.2.0.eb]: We should use the http here as well

Comment (Darkless012) [reply easybuild/easyconfigs/f/FastTree/FastTree-2.1.11-GCCcore-8.2.0.eb]: done

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3114.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/1630b6fc8694b0cd016ccef2d81809f4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2644.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/464707316f054a823dd953ab97fe2d01 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/93e71a59a06b70447735d5b93c294b6b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Darkless012!

**Pull request merged**